ui/README.md

```diff
@@ -2,12 +2,32 @@
 
 FusorUi is a rails engine that is added to [Foreman](https://github.com/theforeman/foreman/).
 
-It is currently just adds assets (javascript, stylesheets, images, fonts) from an ember-cli project entitled [fuser-demo](https://github.com/fusor/fusor-demo/)
+Fusor_ui is essentially the output from the /dist directory of [https://github.com/fusor/fusor-ember-cli/](https://github.com/fusor/fusor-ember-cli/) which is an [ember-cli](http://www.ember-cli.com/) project that contains assets such as javascript, stylesheets, images, and fonts.
+
+## Important Note
+
+Do NOT work directly on the files in [app/assets](https://github.com/fusor/fusor/tree/master/ui/app/assets). These files will be overwritten by future commits when the code in [fusor-ember-cli](https://github.com/fusor/fusor-ember-cli/) changes.
+
+The fusor-ember-cli/dist directory is listed in the [.gitignore](https://github.com/fusor/fusor/blob/master/.gitignore) file of [fusor](https://github.com/fusor/fusor/) so you wont find it in the [fusor-ember-cli](https://github.com/fusor/fusor-ember-cli/) repository.
+
+The /dist distory is generated automatically by [ember-cli](http://www.ember-cli.com/) if you run `ember server` or `ember build` inside the locally or inside the [fusor-ember-cli](https://github.com/fusor/fusor-ember-cli/) directory.
+
+Follow the workflow below for more steps.
+
+## Workflow
+
+1. Clone [fusor](https://github.com/fusor/fusor/).
+2. `cd fusor-ember-cli`
+3. run `ember server`
+4. `cd ../ui`
+6. run script `./copy-fusor-ember-cli-to-assets` which copies files from `fusor-ember-cli/dist` to the `fusor/ui` repo
+7. `cd ../` to return to local `fusor` project root directory
+8. git push changes to your forked repo of fusor and create PR
```

Comment:
The workflow is confusing for me :/

I am expecting to run a rails server and get any change I make in the ember ui files. This seems to suggest that I need to run ember server to get the changes immediately but without rest of the rails application. If this is true can it be fixed? Symlink instead of copy or mapping a route from rails server to ember server or something else ?